### PR TITLE
Rating & Snackbar coomponents

### PR DESCRIPTION
**Pull request by HarshnaNHaswani**
https://user-images.githubusercontent.com/82398518/154738402-40663382-08d2-49ec-9f1a-350803ba04b8.mp4

### CODE REVIEW

File: pages/components/rating.html [hunk 1]
```diff
@@ -106,8 +106,35 @@ <h1 clss="page-title">Ratings</h1>
 
 
   <main class="main-content">
-
-    <iframe src="/codeblock/x.txt" frameborder="0" class="codeblock"></iframe>
+    <div class="rating-wrapper wrap-color">
+        <input type="radio" name="rating" id="star-1">
+        <label for="star-1">✫</label>
```
Comment (mohitdhatrak) [inline pages/components/rating.html:111]: not sure about this, but i think using an icon for star would be better than typing it? I guess it's fine maybe

Comment (HarshnaNHaswani) [reply pages/components/rating.html:111]: Icons may take time to load. this is unicode value like &nbsp; or &gt;

File: pages/components/snackbar.html [hunk 2]
```diff
@@ -116,6 +145,6 @@ <h1 clss="page-title">Snackbar</h1>
     <p>Developed by Harshna Haswani</p>
   </footer>
```
Comment (mohitdhatrak) [inline pages/components/snackbar.html:146]: footer opening tag missing?

File: styles/06-components/brand-theme.css [hunk 3]
```diff
@@ -1,6 +1,7 @@
 body {
   color: var(--color-text);
   background-color: var(--color-background);
+  min-height: 100vh;
 }
 
 .nav-main {
```
Comment (mohitdhatrak) [inline styles/06-components/brand-theme.css:7]: forgot to remove this probably, it has no CSS closing bracket too

File: pages/components/snackbar.html (hunk 2 above)
Comment (HarshnaNHaswani) [reply pages/components/snackbar.html:146]: no it is not
click the bright blue on the left

File: styles/06-components/rating.css [hunk 4]
```diff
@@ -0,0 +1,42 @@
+:root {
+  --color-rating-before: rgb(126, 125, 125);
+  --color-rating-after: #e7d10a;
+}
```
Comment (mohitdhatrak) [inline styles/06-components/rating.css]: some places you used hex code and some places rgba and in borders color names, so to maintain one pattern stick to only 1 of these.

File: styles/06-components/brand-theme.css (hunk 3 above)
Comment (HarshnaNHaswani) [reply styles/06-components/brand-theme.css:7]: hi, same as above, please press the ... on the left

File: styles/06-components/snackbar.css [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  display: flex;
+  align-items: center;
+  padding: 0 10px;
+  position: fixed;
+  bottom: 2%;
+  left: 50%;
+  transform: translateX(-50%);
+  border-radius: 10px;
+  font-weight: 200;
+  z-index: 99999;
```
Comment (mohitdhatrak) [inline styles/06-components/snackbar.css:11]: why such a large value? i think smaller values would do too

Comment (HarshnaNHaswani) [reply styles/06-components/snackbar.css:11]: to prevent any chance of override

**Review by mohitdhatrak — COMMENTED** [review]
Gave some inputs, have a look. Good work!

**Comment by HarshnaNHaswani** [comment]
Thank you @mohitdhatrak for your time and pointing things out

File: styles/06-components/rating.css [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+.rating-wrapper{
+  direction: rtl;
+  text-align: center;
+  position: relative;
+  cursor: pointer;
+  margin: 5ch;
+}
+
+.rating-wrapper input[type="radio"] {
```
Comment (irajbhinde) [inline styles/06-components/rating.css:14]: is this being used to hide the radio button circle?

File: styles/06-components/snackbar.css [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+  width: max-content;
+}
+
+.snackbar *{
```
Comment (irajbhinde) [inline styles/06-components/snackbar.css:15]: why use * after assigning a style to a class, shouldn't it automatically apply to all its elements?

**Review by irajbhinde — COMMENTED** [review]
The naming convention for the class is very clear & easy to understand, kudos for that. It would be helpful to give a better review if you could preview deploy your work

File: styles/06-components/rating.css (hunk 6 above)
Comment (HarshnaNHaswani) [reply styles/06-components/rating.css:14]: Yes

File: styles/06-components/snackbar.css (hunk 7 above)
Comment (HarshnaNHaswani) [reply styles/06-components/snackbar.css:15]: e.g. If I just  do snackbar padding 2ch. It'll give the padding to the snackbar container. But if I give it this way, every element inside the snackbar will have a padding and not the container itself

Comment (HarshnaNHaswani) [reply styles/06-components/snackbar.css:15]: I had a padding there earlier so it made sense, now though its redundant and not useful

**Comment by HarshnaNHaswani** [comment]
Thank you @irajbhinde for your time and insights

**Pull request merged**